.github/workflows/ci.yaml

```
@@ -51,6 +52,12 @@ jobs:
       - run: make go-mod-tidy
       - run: make BUILDTAGS=kerneldrv
       - run: make test BUILDTAGS=kerneldrv
+      - name: Install envtest tool and run envtest
+        run: |
+          go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+          setup-envtest use ${K8S_VERSION}
+          source <(setup-envtest use -p env ${K8S_VERSION})
+          make envtest
```

not a big deal but I was thinking to keep install and run separate, i.e. have `- name: Install..` a bit earlier and then just:

```suggestion
      -run: make envtest
```

---

Umm, I changed as you suggested, but there is ci error, I guess the last two lines have to be together.!